aspnetcore/release-notes/aspnetcore-6.0/samples/HomeController.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System.Diagnostics;
+using System.Text.Json;
+using WebAsync.Models;
+
+namespace WebAsync.Controllers
+{
+    #region snippet
+    public class HomeController : Controller, IAsyncDisposable
```

Comment:
@pranavkm  quick review of IAsyncDisposable
Can you take a look at the [IAsyncDisposable writeup here](https://review.docs.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-6.0?view=aspnetcore-6.0&branch=pr-en-us-23640#iasyncdisposable-supported)?